addons/l10n_fr/models/res_company.py

```diff
@@ -20,6 +23,13 @@ def _is_accounting_unalterable(self):
             return False
         return self.country_id and self.country_id.code in self._get_unalterable_country()
 
+    @api.constrains('ape')
+    def _check_ape_code(self):
+        for company in self:
+            if company.ape\
+                    and not re.match(r'\d{4}[A-Z]', company.ape):
```

Comment:
```suggestion
            if company.ape and not re.match(r'\d{4}[A-Z]', company.ape):
```

https://peps.python.org/pep-0008/#maximum-line-length
> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation. 

Also, don't use a ridiculously small max line length :pray:

Reply:
Indeed, I don't know why I put some line return here :thinking: Thanks!